docs/manage/data/permissions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Data Management Permissions"
+linkTitle: "Permissions"
+description: "Data management permissions."
+weight: 50
+type: "docs"
+tags: ["data management", "cloud"]
+# SME: Devin Hilly
+---
+
+When you view the [**DATA** tab](https://app.viam.com/data/view) while authenticated to an {{< glossary_tooltip term_id="organization" text="organization" >}}, you see data from all resources you are an owner of within that organization.
+This includes data from any resources of locations shared with organizations you own.
+
+Users have different capabilities when it comes to managing data, depending on whether they are [an owner or an operator](/manage/fleet/#permissions) of a given {{< glossary_tooltip term_id="organization" text="organization" >}}, {{< glossary_tooltip term_id="location" text="location" >}}, or {{< glossary_tooltip term_id="robot" text="robot" >}}.
+The following table summarizes the permissions for owners and operators at different levels:
+
+| Permissions                         | Org owner | Org operator | Location owner                                | Location operator | Robot owner                                   | Robot operator |
+| ----------------------------------- | --------- | ------------ | --------------------------------------------- | ----------------- | --------------------------------------------- | -------------- |
+| View data                           | **Yes**   | No           | **Yes\***                                     | No                | **Yes\*\***                                   | No             |
+| See data tags                       | **Yes**   | No           | Only tags applied to data they have access to | No                | Only tags applied to data they have access to | No             |
+| Edit data (add tags, delete info)   | **Yes**   | No           | **Yes\***                                     | No                | **Yes\*\***                                   | No             |
+| Train models                        | **Yes**   | No           | **Yes** on data they have access to           | No                | **Yes** on data they have access to           | No             |
+| Upload models/packages              | **Yes**   | No           | **Yes\***                                     | No                | **Yes**                                       | No             |
```

Comment:
I don't think we need an asterisk next to the Yes in the location owner column.

Also, I like how on lines 24 and 25 it uses the word "organization" to refer to models and packages. Maybe we can add that on lines 23 and 26 as well. Models and packages are scoped to the organization so I think the explicit callout is nice - that the user is getting access to upload/view/use something at the organization level, even if they only have owner access on a single resource within that organization.

Reply:
all of the above done!